hotel-api-persistence/src/main/java/com/hotelapp/persistence/impl/HotelRepositoryImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.hotelapp.persistence.impl;
+
+import com.hotelapp.persistence.HotelRepository;
+
+import javax.inject.Inject;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.UUID;
+
+public class HotelRepositoryImpl implements HotelRepository {
```

Comment:
Are we handling any kind of  transactionality in this service?

Reply:
Since I have only added the createHotel method, there is no transactionality yet, as per Java Doc Definition "A transaction is a set of one or more statements that is executed as a unit" (https://docs.oracle.com/javase/tutorial/jdbc/basics/transactions.html)
But, it definitely will be necessary to handle transactionality. One use case that comes to my mind is when creating a reservation, that requires changes in some tables.